damnit/gui/plot.py

```diff
@@ -542,8 +556,8 @@ def _button_plot_clicked(self, runs_as_series):
                 try:
                     correlated, xi, yi = self.get_run_series_data(p, r, xlabel, ylabel)
                     strongly_correlated = strongly_correlated and correlated
-                except Exception:
-                    pass
+                except Exception as e:
+                    log.warning(f"Couldn't retrieve data for '{xlabel}', '{ylabel}': {e}")
```

Comment:
```suggestion
                    log.warning(f"Couldn't retrieve data for '{xlabel}', '{ylabel}'", exc_info=True)
```

This should give us a traceback in the logging.